workshops/sprig_dodge/README.md

```diff
@@ -0,0 +1,511 @@
+---
+name: 'Sprig Dodging Game'
+description: 'Doge falling obstacles. Learn how to make it using Sprig!'
+author: '@SamDev-7'
+img: 'https://cloud-ijhz85il3-hack-club-bot.vercel.app/0image.png'
+---
+
+In this workshop, we'll be building a dodging game using [Sprig](https://sprig.hackclub.com). The player will have to doge the falling obstacles to score points. Sprig is a JavaScript game engine that makes it easy for beginners and masters to make games, all in your browser.
+
+Basic JavaScript knowledge is recommended for this workshop.
+If you're not familiar with Sprig, you can check out the [Getting Started Guide](https://github.com/hackclub/sprig/blob/main/docs/GETTING_STARTED.md) to learn the basics.
```

Comment:
Maybe this could be rephrasing into recommending readers do https://workshops.hackclub.com/sprig_first_game/ first

Reply:
Hopefully it's better now :)